test/ng/directive/inputSpec.js

```diff
@@ -3046,6 +3046,26 @@ describe('input', function() {
       expect(inputElm[0].getAttribute('value')).toBe('something');
     });
 
+    they('should update the $prop "value" property and attribute after change the "value" property', {
+      input: '<input type="text" ng-value="value">',
+      textarea: '<textarea ng-value="value">></textarea>'
+    }, function(tmpl) {
+      var element = helper.compileInput(tmpl);
+
+      $rootScope.$apply(function() {
+        $rootScope.value = 'something';
+      });
+      expect(element[0].value).toBe('something');
+      expect(element[0].getAttribute('value')).toBe('something');
+
+      helper.changeInputValueTo('newValue');
```

Comment:
Yes, you should add an expect. Otherwise you can't be sure that the input / scope has really been changed.